test/validators.test.js

```diff
@@ -6860,6 +6860,24 @@ describe('Validators', () => {
           '0114152198',
         ],
       },
+      {
+        locale: 'ar-SD',
+        valid: [
+          '0128652312',
+          '+249919425113',
+          '249123212345',
+          '0993212345',
+        ],
+        invalid: [
+          '12345',
+          '',
+          '+249962662622',
```

Comment:
This phone number is not invalid due to the regular expression so that's why the test is failing